app/battle.py

```diff
@@ -0,0 +1,25 @@
+from typing import List

```

Comment:
why not just use usual list annotation?

Reply:
I want to specify the list with the passed class instances explicitly. I think it's appropriate here. But I have corrected this annotation in other files